### DISCUSSION TITLE
Reading a group address via device node in universall mode always reads all group addresses

### DISCUSSION THREAD

**Discussion by Mike-FUT** [Supergiovane/node-red-contrib-knx-ultimate#304] [Q&A] [answered]
I accidentally created a KNX bus spam machine by reading some addresses from KNX from a device node in universal node.

According to documentation I'm providing the group address as "msg.topic" and I'm sending "true" as "msg.readstatus" (as one object of course).
But that causes a massive load by reading ALL bus. It seems like msg.topic is simply ignored. Do I have to use a different variable for reading instead of msg.topic?

**Comment by Supergiovane** [comment, marked as answer]
Hi
With the node set in "universal mode", you must use
msg.destination="0/0/26"
msg.readstatus=true
Can you please try with that?

**Comment by Mike-FUT** [comment]
Oh dear, how embarrassing. Seems like I had a complete blackout considering I have been reading the docs multiple times and they clearly say to use msg.destination.
Apologies and many thanks! That was of course the solution!
